unit/src/PorousFlowEffectiveSaturationRSCTest.C

```diff
@@ -0,0 +1,59 @@
+/****************************************************************/
+/*               DO NOT MODIFY THIS HEADER                      */
+/* MOOSE - Multiphysics Object Oriented Simulation Environment  */
+/*                                                              */
+/*           (c) 2010 Battelle Energy Alliance, LLC             */
+/*                   ALL RIGHTS RESERVED                        */
+/*                                                              */
+/*          Prepared by Battelle Energy Alliance, LLC           */
+/*            Under Contract No. DE-AC07-05ID14517              */
+/*            With the U. S. Department of Energy               */
+/*                                                              */
+/*            See COPYRIGHT for full restrictions               */
+/****************************************************************/
+#include "PorousFlowEffectiveSaturationRSCTest.h"
+
+CPPUNIT_TEST_SUITE_REGISTRATION( PorousFlowEffectiveSaturationRSCTest );
+
+PorousFlowEffectiveSaturationRSCTest::PorousFlowEffectiveSaturationRSCTest()
+{
+  _ep = 1.0E-8;
+}
+
+PorousFlowEffectiveSaturationRSCTest::~PorousFlowEffectiveSaturationRSCTest()
+{}
```

Comment:
brackets should be on separate lines. I'n most cases an empty virtual destructor is unnecessary. It is only needed in the top most base classes.